src/main/java/seedu/address/storage/JsonAdaptedProperty.java

```diff
@@ -8,46 +8,41 @@
 
 import com.fasterxml.jackson.annotation.JsonCreator;
 import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
 
 import seedu.address.commons.exceptions.IllegalValueException;
-import seedu.address.model.field.Email;
 import seedu.address.model.field.Name;
-import seedu.address.model.field.Phone;
+import seedu.address.model.field.Person;
 import seedu.address.model.field.Price;
 import seedu.address.model.property.Address;
 import seedu.address.model.property.Property;
-import seedu.address.model.property.Seller;
 import seedu.address.model.tag.Tag;
 
 /**
  * Jackson-friendly version of {@link Property}.
  */
+@JsonPropertyOrder({"name", "address", "seller", "price", "tagged" })
 class JsonAdaptedProperty {
 
     public static final String MISSING_FIELD_MESSAGE_FORMAT = "Property's %s field is missing!";
 
     private final String name;
-    private final String seller;
-    private final String phone;
-    private final String email;
     private final String address;
+    private final JsonAdaptedPerson seller;
     private final String price;
     private final List<JsonAdaptedTag> tagged = new ArrayList<>();
 
     /**
      * Constructs a {@code JsonAdaptedProperty} with the given property details.
      */
     @JsonCreator
-    public JsonAdaptedProperty(@JsonProperty("name") String name, @JsonProperty("phone") String phone,
-                               @JsonProperty("email") String email, @JsonProperty("address") String address,
-                               @JsonProperty("tagged") List<JsonAdaptedTag> tagged,
-                               @JsonProperty("seller") String seller, @JsonProperty("price") String price) {
+    public JsonAdaptedProperty(@JsonProperty("name") String name, @JsonProperty("address") String address,
+                               @JsonProperty("phone") JsonAdaptedPerson seller, @JsonProperty("price") String price,
```

Comment:
```suggestion
                               @JsonProperty("seller") JsonAdaptedPerson seller, @JsonProperty("price") String price,
```

Reply:
Nice catch. I wonder why this mistake does not show in the json...